### PR TITLE
Create Devanshkanda.json

### PR DESCRIPTION
**Pull request by Devanshkanda**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6692"></a>

### CODE REVIEW

**Review by Aniket1026 — CHANGES_REQUESTED** [review]
Thanks for contributing.

Please move your file to the data folder.
You can have a look at QuickStart Guide (especially section 7) in Docs: https://linkfree.io/docs/quickstart to help you update this Pull Request.

File: Devanshkanda.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Devansh kanda",
+  "displayStatsPublic": true,
```
Comment (khushalarora) [inline Devanshkanda.json:3]: You can remove this line, it is not required anymore :+1: 
```Suggestion
```

File: Devanshkanda.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "type": "personal",
+  "bio": "i am a Tech Geek, a Software Developer with Security Mindset",
+  "avatar": "https://github.com/Devanshkanda.png",
```
Comment (khushalarora) [inline Devanshkanda.json:6]: You can also remove this line, it is not required anymore :+1: 
```Suggestion
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hey @Devanshkanda, thank you for contributing :tada: 
In addition to above comment, you can also make below changes :+1:

File: Devanshkanda.json (hunk 2 above)
Comment (Devanshkanda) [reply Devanshkanda.json]: Thanks @khushalarora for recommending changes, I have updated the file path and the changes you suggested.

**Pull request merged**